CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.27.0
+
+* Add support for nested maps in `map.has-key`.
```

Comment:
This should also link to the docs.

Reply:
Will do. I'm going to wait until #1076 is merged